browser/mapping_test.go

```diff
@@ -343,6 +343,7 @@ type pageAPI interface {
 	SetViewportSize(viewportSize goja.Value)
 	Tap(selector string, opts goja.Value)
 	TextContent(selector string, opts goja.Value) string
+	throttleNetwork(common.NetworkProfile) error
```

Comment:
There is actually no need for this to be capitalized, but considering that the other methods follow this format, should this one follow that pattern too?

Reply:
Nice spot, i think it's best to be consistent, i've changed it in 20e9ca5cbda25f7844aab89636bbf78c7b0276b2